test-fork/peripheral/AuraMining.spec.ts

```diff
@@ -0,0 +1,79 @@
+import { BN, simpleToExactAmount } from "../../test-utils/math";
+import { ethers, network } from "hardhat";
+import { expect } from "chai";
+import { AuraMining, AuraMining__factory } from "../../types/generated";
+import { impersonateAccount, ZERO_ADDRESS } from "../../test-utils";
+import { Signer } from "ethers";
+import { config } from "../../tasks/deploy/mainnet-config";
+import { Phase2Deployed } from "scripts/deploySystem";
+import { Account } from "types/common";
+
+const EMISSIONS_MAX_SUPPLY = 50000000;
+const EMISSIONS_INIT_SUPPLY = 50000000;
+const ALCHEMY_API_KEY = process.env.NODE_URL;
+
+describe("AuraMining", () => {
+    let cvxMining: AuraMining;
+    let signer: Signer;
+    let phase2: Phase2Deployed;
+    let operatorAccount: Account;
+    const aliceAddress = "0x3000d9B2c0E6B9F97f30ABE379eaAa8A85A04afC";
```

Comment:
Can probably just use `accounts[0]` from `getSigners` rather than keep using this addy. Keep things generic.